src/huggingface_hub/inference/_client.py

```diff
@@ -773,6 +773,51 @@ def text_to_speech(self, text: str, *, model: Optional[str] = None) -> bytes:
         response = self.post(json={"inputs": text}, model=model, task="text-to-speech")
         return response.content
 
+    def zero_shot_image_classification(
+        self, image: ContentT, labels: List[str], *, model: Optional[str] = None
+    ) -> List[ClassificationOutput]:
+        """
+        Provide input image and text labels to predict text labels for the image.
+
+        Args:
+            image (`Union[str, Path, bytes, BinaryIO]`):
+                The input image to caption. It can be raw bytes, an image file, or a URL to an online image.
+            labels (`List[str]`):
+                List of string possible labels. The `len(labels)` must be greater than 1.
+            model (`str`, *optional*):
+                The model to use for inference. Can be a model ID hosted on the Hugging Face Hub or a URL to a deployed
+                Inference Endpoint. This parameter overrides the model defined at the instance level. Defaults to None.
+
+        Returns:
+            `List[Dict]`: List of classification outputs containing the predicted labels and their confidence
+
+        Raises:
+            [`InferenceTimeoutError`]:
+                If the model is unavailable or the request times out.
+            `HTTPError`:
+                If the request fails with an HTTP error status code other than HTTP 503.
+
+        Example:
+        ```py
+        >>> from pathlib import Path
+        >>> from huggingface_hub import InferenceClient
+        >>> client = InferenceClient()
+
+        >>> client.zero_shot_image_classification("https://upload.wikimedia.org/wikipedia/commons/thumb/4/43/Cute_dog.jpg/320px-Cute_dog.jpg", "dog,cat,horse", model="openai/clip-vit-base-patch32")
+        [{"label": "dog", "score": 0.956}, ...]
+        """
+
+        # Raise valueerror if input is less than 2 labels
+        if len(labels) < 2:
+            raise ValueError("You must specify at least 2 classes to compare. Please specify more than 1 class.")
+
+        response = self.post(
+            json={"image": _b64_encode(image), "parameters": {"candidate_labels": labels}},
```

Comment:
```suggestion
            json={"image": _b64_encode(image), "parameters": {"candidate_labels": ",".join(labels)}},
```
Since labels is now a `List[str]` it must be converted to a comma-separated string